package.json

```diff
@@ -85,9 +85,9 @@
     "webpack-merge": "^4.1.2"
   },
   "build": {
-    "appId": "com.nos.electron",
+    "appId": "io.nos",
```

Comment:
Should we leave this with `electron` on it?  Or maybe more appropriately `desktop`?  i.e.: `io.nos.desktop`